test/multiarch_utils.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import "runtime"
+
+var (
+	imageNames    = initImageNames()
+	excludedTests = initExceludedTests()
+)
+
+const (
+	// Busybox image with specific sha
+	BusyboxSha = iota
+	// Registry image
+	Registry
+)
+
+func initImageNames() map[int]string {
+	mapping := make(map[int]string)
+
+	switch arch := runtime.GOARCH; arch {
+	case "s390x":
+		mapping[BusyboxSha] = "busybox@sha256:4f47c01fa91355af2865ac10fef5bf6ec9c7f42ad2321377c21e844427972977"
+		mapping[Registry] = "ibmcom/registry:2.6.2.5"
+
+	default:
+		mapping[BusyboxSha] = "busybox@sha256:895ab622e92e18d6b461d671081757af7dbaa3b00e3e28e12505af7817f73649"
+		mapping[Registry] = "registry"
+	}
+	return mapping
+}
+
+func initExceludedTests() map[string]bool {
```

Comment:
typo: `initExcludedTests`

Reply:
Also, not sure why the `init` prefix :stuck_out_tongue_closed_eyes:

Reply:
thanks for the catch! fixed it
init is used because the func initiates list of excluded tests. Do you think it better to use another name?